backend/src/members/members.controller.ts

```diff
@@ -15,6 +15,13 @@ import { GetMember, MembersService, PostMember } from './members.service';
 export class MembersController {
   constructor(private readonly memberService: MembersService) {}
 
+  @UseGuards(AuthGuard)
+  @Get()
+  async getMembers(): Promise<GetMember[]> {
+    const memebers = await this.memberService.findAll();
+    return memebers.map((getMember) => this.memberService.mapMemeberToMemberDto({ memeber: getMember }));
+  }
```

Comment:
I suggest this can be "simplified" a bit

`return members.map((member) => this.memberService.mapMemberToMemberDto(member));`